src/app/core/forms/notification-channel.form.ts

```diff
@@ -0,0 +1,42 @@
+import { FormControl, FormGroup } from '@angular/forms';
+import {NotificationChannelTypesEnum} from '../enums/notification-channel-types.enum';
+
+export class NotificationChannel extends FormGroup {
+  constructor() {
+    super({
+      ID: new FormControl(),
+      Name: new FormControl(),
+      Type: new FormControl(),
+      Recipients: new FormControl(),
+      WebhookURL: new FormControl()
```

Comment:
To powinno być wymagane jeśli type == 'webhook'

Reply:
done